gfx/gfx_thumbnail_path.c

```diff
@@ -555,6 +566,173 @@ bool gfx_thumbnail_set_content_playlist(
    return true;
 }
 
+bool gfx_thumbnail_set_icon_playlist(
+      gfx_thumbnail_path_data_t *path_data, playlist_t *playlist, size_t idx)
+{
+   const char *content_path           = NULL;
+   const char *content_label          = NULL;
+   const char *core_name              = NULL;
+   const char *db_name                = NULL;
+   const struct playlist_entry *entry = NULL;
+   char content_dir[PATH_MAX_LENGTH];
+   const char *dir_thumbnails = NULL;
+   settings_t *settings       = config_get_ptr();
+
+   if (!path_data)
+   {
+      return false;
+   }
+
+
+   /* When content is updated, must regenerate icon
+    * thumbnail paths */
+   path_data->icon_path[0]           = '\0';
+
+   /* 'Reset' path_data content strings */
+   path_data->content_path[0]         = '\0';
+   path_data->content_label[0]        = '\0';
+   path_data->content_core_name[0]    = '\0';
+   path_data->content_db_name[0]      = '\0';
+   path_data->content_img[0]          = '\0';
+   path_data->content_img_full[0]     = '\0';
+   path_data->content_img_short[0]    = '\0';
+
+   /* Must also reset playlist thumbnail display modes */
+   path_data->playlist_icon_mode     = PLAYLIST_THUMBNAIL_MODE_DEFAULT;
+   path_data->playlist_index          = 0;
+
+   if (!playlist)
+   {
```

Comment:
For single lines, the brackets are not necessary

Reply:
code reformatted